plugins/search-react/src/components/SearchTracker/SearchTracker.tsx

```diff
@@ -23,14 +23,15 @@ import { useSearch } from '../../context';
  */
 export const TrackSearch = ({ children }: { children: React.ReactChild }) => {
   const analytics = useAnalytics();
-  const { term } = useSearch();
+  const { term, result } = useSearch();
+  const numberOfResults = result.value?.results.length ?? 0;
 
   useEffect(() => {
     if (term) {
       // Capture analytics search event with search term provided as value
-      analytics.captureEvent('search', term);
+      analytics.captureEvent('search', term, { value: numberOfResults });
```

Comment:
I'm wondering if the `numberOfResults`, might be more appropriate as an attribute?

e.g.

```ts
analytics.captureEvent('search', term, { attributes: { numberOfResults } });
```